### PR TITLE
Update IREE to 20220811.232

### PR DESCRIPTION
**Pull request by rsuderman**
"iree-compiler": "20220809.230",
"iree-runtime": "20220809.230",

### CODE REVIEW

File: version_info.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "pinned-versions": {
-    "iree-compiler": "20220726.216",
-    "iree-runtime": "20220726.216",
-    "iree-tools-xla": "20220726.216",
+    "iree-compiler": "20220809.230",
+    "iree-runtime": "20220809.230",
+    "iree-tools-xla": "20220809.230",
     "jax": "0.3.15",
```
Comment (GMNGeoffrey) [inline version_info.json:6]: Can we bump this to latest as well? Looks like 0.3.16 is on PyPi now, but jaxlib still only has 0.3.15. What's up with that?

image
image

File: version_info.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "pinned-versions": {
-    "iree-compiler": "20220726.216",
-    "iree-runtime": "20220726.216",
-    "iree-tools-xla": "20220726.216",
+    "iree-compiler": "20220809.230",
```
Comment (GMNGeoffrey) [inline version_info.json:3]: We're looking to promote [20220811.232](https://github.com/iree-org/iree/releases/tag/candidate-20220714.204). Can we use that instead?

**Review by GMNGeoffrey — APPROVED** [review]
Maybe the commit title could be more specific to aid in searches through history?

"Update IREE to 20220811.232"

Especially since you're *also* updating iree-tools-xla, not just iree-compiler and iree-runtime

File: version_info.json (hunk 2 above)
Comment (rsuderman) [reply version_info.json:3]: Sadly no. This is the most recent version still compatible with 0.3.15.

File: version_info.json (hunk 1 above)
Comment (rsuderman) [reply version_info.json:6]: I was hoping to test 0.3.16 with the auto-update tool.

**Comment by GMNGeoffrey** [comment]
Ah well now the description is just incorrect... the suggestion I gave was for if you picked the later version

File: version_info.json (hunk 2 above)
Comment (jpienaar) [reply version_info.json:3]: Is this due to attribute changes?

**Pull request merged**